firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -15,17 +15,16 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
         dataClearanceContextHintVC.stopTimer()
     }
 
-    func configureDataClearanceContextualHint() {
-        guard !isToolbarRefactorEnabled,
-                contentContainer.hasWebView,
+    func configureDataClearanceContextualHint(_ view: UIView) {
+        guard contentContainer.hasWebView,
                 tabManager.selectedTab?.url?.displayURL?.isWebPage() == true
         else {
             resetDataClearanceCFRTimer()
             return
         }
         dataClearanceContextHintVC.configure(
-            anchor: navigationToolbar.multiStateButton,
-            withArrowDirection: topTabsVisible ? .up : .down,
+            anchor: view,
+            withArrowDirection: ToolbarHelper().shouldShowNavigationToolbar(for: traitCollection) ? .down : .up,
```

Comment:
Is there a reason we don't check for `ToolbarHelper().shouldShowTopTabs(for: traitCollection)` instead here? The old code was checking for top tabs.

Reply:
I think it's ok to leave it like this. I don't see it causing any issues with the old toolbar.